### DISCUSSION TITLE
post/cilium-egress-gateway/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#96] [Announcements]
# post/cilium-egress-gateway/

카카오페이증권 Gateway에 Cilium egress gateway 활용한 경험을 공유합니다.

https://tech.kakaopay.com/post/cilium-egress-gateway/

**Comment by anfrhrl5555** [comment]
안녕하세요 cilium-egress-gateway에 관한 심도있는 지식 얻고 갑니다.

한 가지 궁금한 게 있는데요, egressGateWay의 Gateway로 설정한 Node가 장애로 인해 죽게 되면, 해당 Node의 GateWayIP로 통신을 할 수 없게 되는데 서비스의 가용성을 어떻게 확보하신 건지 궁금합니다.

**Reply by anfrhrl5555** [reply]
추가로 multus, wherabout같은 오픈소스를 사용하지 않고, Pod에 ip addr로 직접 추가한 건 오픈소스의 불안정성 및 Pod에 직접 IP를 설정하므로써 안정성 확보가 목표인걸까요?

**Reply by anfrhrl5555** [reply]
위의 예제에서는 "kubernetes.io/hostname: node01.hostname" << 라벨이 설정된 Node가 종료된 경우라고 볼 수 있겠습니다.
